src/main/java/seedu/address/logic/commands/ListEntryCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.entry.ListEntryFormatPredicate;
+
+/**
+ * Lists all entries in address book based on the filter provided
+ */
+public class ListEntryCommand extends Command {
+    public static final String COMMAND_WORD = "elist";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all entries "
+            + "by displaying them as a list sorted by date. Entries can also be listed by day/week.\n"
+            + "Possible parameters: day/week \n"
```

Comment:
There's the option to just use elist without the parameters also right? Perhaps can update the message here